pages/community/community-contribute-to-precice.md

```diff
@@ -71,23 +68,26 @@ First time working with Git? Watch a [short lecture on the GitHub workflow](http
 Our tutorials generally follow a file structure similar to this:
```

Comment:
Should we somehow point out which files are optional and which ones are mandatory?

Reply:
Good point, it could make the list easier to parse. Which ones would you classify as mandatory and which as optional?

Reply:
I split the bullet list into sections. This hopefully also helps to read this page as a helpful step-by-step guide and not just as a list of arbitrary bureaucratic laws.

Reply:
We could point out the optional once explicitly: 

```
<tutorial>/                       # e.g. perpendicular-flap/
  - README.md                     # description of the case
  - precice.config.xml            # a works-with-all preCICE configuration file
  - clean-tutorial.sh             # a symbolic link (see ../tools/)
  - <visualization scripts>       # (optional) gnuplot or simple Python scripts
  - images/                       # (optional) any images used by the documentation
  - solver-<code>/                # (optional) any configurable, tutorial-specific code, e.g., solver-fenics
  - <participant1-solver1>/       # e.g. fluid-openfoam/
    - run.sh                      # a short script to run the solver1 case
    - clean.sh                    # a short script to clean the solver1 case
    - <the solver1 files>
  - <participant2-solver2>/       # e.g. solid-dealii/
    - run.sh
    - clean.sh
    - <the solver2 files>
  - reference-results/            # (optional) results from different case combinations, used for regression tests
    - <case_combination>.tar.gz
```

Reply:
Ah, of course, this list was more "these are all the elements you may encounter", not "these are all things you need to provide". I will split it in two parts.